packages/components/src/vue/components/word-list/word-language-panel.vue

```diff
@@ -293,7 +303,47 @@ export default {
           context: Object.keys(wordItem.formattedContext).join(' ')
         }
       })
-      const result = Download.collectionToCSV(';', exportFields)(wordlistData)
+
+      return {
+        exportFields, wordlistData, delimiter: ';'
+      }
+    },
+
+    prepareDownloadListQuizlet () {
+      const exportFields = [ 'word', 'definition']
+      const source = this.downloadWithFilter ? this.wordItems : this.wordlist.values
+
+      let wordlistData = []
+      source.forEach(wordItem => {
+        if (wordItem.homonym && wordItem.homonym.lexemes) {
+          wordItem.homonym.lexemes.forEach(lexeme => {
+            if (lexeme.hasShortDefs) {
+              lexeme.meaning.shortDefs.forEach(shortDef => {
+                wordlistData.push({
+                  word: `${wordItem.homonym.targetWord} (${lexeme.lemma.word})`,
```

Comment:
see comments in #200. I would like to include the principal parts with the lemma.